masterlist.yaml

```diff
@@ -125,6 +122,8 @@ plugins:
   - name: 'Anchorage.esm'
     url: [ 'http://taleoftwowastelands.com/']
     priority: -400998000
+    after:
+      - 'Fallout3.esm'
```

Comment:
That shouldn't be necessary. The masterlist should have as few `after`s as possible to let LOOT do its own magic. As long as `Anchorage.esm` has `Fallout3.esm` as a master, it should be placed properly.